### PR TITLE
Added Side Navigation with route navigating to blank components

### PR DESCRIPTION
**Pull request by meet-aniket**

### CODE REVIEW

**Comment by meet-aniket** [comment]
@azkyakhan

File: changelogs/unreleased/-1vju5j1.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Added Side Navigation with route navigating to blank components (#1vju5j1)
```
Comment (ymaheshwari1) [inline changelogs/unreleased/-1vju5j1.yml:2]: Remove ticket id from the title

File: src/App.vue [hunk 2]
```diff
@@ -1,42 +1,46 @@
...
-    IonRouterOutlet
+    IonRouterOutlet,
+    IonSplitPane,
+    Menu,
   },
   data() {
     return {
-      loader: null as any
-    }
+      loader: null as any,
```
Comment (ymaheshwari1) [inline src/App.vue:27]: Remove the auto-formatting changes

File: src/components/Menu.vue [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            router-direction="root"
+            :router-link="menu.url"
+            class="hydrated"
+            :class="{ selected: selectedIndex === index }"
+          >
+            <ion-icon
+              slot="start"
+              :ios="menu.iosIcon"
+              :md="menu.mdIcon"
+            ></ion-icon>
```
Comment (ymaheshwari1) [inline src/components/Menu.vue:31]: Use a self-closing tag.

File: src/components/Menu.vue [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  IonContent,
+  IonIcon,
+  IonHeader,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonTitle,
+  IonToolbar,
+  IonMenu,
+  IonMenuToggle,
```
Comment (ymaheshwari1) [inline src/components/Menu.vue:51]: Arrange the import alphabetically.

File: src/router/index.ts [hunk 5]
```diff
@@ -1,54 +1,68 @@
...
 import { RouteRecordRaw } from 'vue-router';
-import Home from '@/views/Home.vue'
-import Login from '@/views/Login.vue'
-import Settings from "@/views/Settings.vue"
-import store from '@/store'
+import Login from '../views/Login.vue';
+import Orders from '../views/Orders.vue';
+import InProgress from '../views/InProgress.vue';
+import Completed from '../views/Completed.vue';
+import Settings from '../views/Settings.vue';
```
Comment (ymaheshwari1) [inline src/router/index.ts:7]: Use @ to declare the file names.

File: src/views/Settings.vue [hunk 6]
```diff
@@ -2,75 +2,119 @@
...
+          interface="popover"
+          :placeholder="$t('store name')"
+          :selected-text="currentFacility.facilityId"
+          @ionChange="setFacility($event)"
+        >
+          <ion-select-option
+            v-for="facility in userProfile.facilities"
+            :key="facility.facilityId"
+            :value="facility.facilityId"
+            >{{ facility.facilityId }}</ion-select-option
```
Comment (ymaheshwari1) [inline src/views/Settings.vue:42]: Use self-closing tag.

File: changelogs/unreleased/-1vju5j1.yml [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+ticket_id: "#1vju5j1"
+merge_request: 16
+author: meet-aniket
```
Comment (adityasharma7) [inline changelogs/unreleased/-1vju5j1.yml:5]: Update this.
Update your git config

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: changelogs/unreleased/-1vju5j1.yml (hunk 7 above)
Comment (meet-aniket) [reply changelogs/unreleased/-1vju5j1.yml:5]: should i update my username with my full name in git config

File: src/router/index.ts [hunk 8]
```diff
@@ -1,7 +1,9 @@
-import { createRouter, createWebHistory } from '@ionic/vue-router';
-import { RouteRecordRaw } from 'vue-router';
-import Home from '@/views/Home.vue'
-import Login from '@/views/Login.vue'
+import { createRouter, createWebHistory } from '@ionic/vue-router'
+import { RouteRecordRaw } from 'vue-router'
+import Login from "@/views/Login.vue"
```
Comment (azkyakhan) [inline src/router/index.ts:3]: Why some imported files are in single quotes and some have double ?

File: src/views/Completed.vue [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  <ion-page>
+    <ion-header>
+      <ion-toolbar>
+        <ion-buttons slot="start">
+          <ion-menu-button></ion-menu-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content :fullscreen="true">
+      wORKING on Completed Page
```
Comment (azkyakhan) [inline src/views/Completed.vue:11]: Improve letter case

File: src/views/Completed.vue [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import { IonButtons, IonHeader, IonMenuButton, IonPage, IonToolbar, IonContent } from "@ionic/vue";
+export default {
+  name: "Completed",
+  components: {
+    IonPage,
```
Comment (azkyakhan) [inline src/views/Completed.vue:21]: Arrange compoenents alphabetically

File: src/views/Completed.vue (hunk 9 above)
Comment (meet-aniket) [reply src/views/Completed.vue:11]: ma'am this is temporary for testing only

File: changelogs/unreleased/-1vju5j1.yml [hunk 11]
```diff
@@ -0,0 +1,6 @@
+---
+title: Added Side Navigation with route navigating to blank components
+ticket_id: "#1vju5j1"
+merge_request:
```
Comment (azkyakhan) [inline changelogs/unreleased/-1vju5j1.yml:4]: MR no. is missing

File: src/components/Menu.vue [hunk 12]
```diff
@@ -0,0 +1,139 @@
+<template>
+  <!-- Application Navigation || Menubar -->
```
Comment (azkyakhan) [inline src/components/Menu.vue:2]: I guess their is no need of this comments

File: src/components/Menu.vue [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+    <ion-header>
+      <ion-toolbar>
+        <ion-title> FULFILLMENT </ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <!-- Menubar Content -->
+    <ion-content>
+      <ion-list>
+        <ion-menu-toggle
```
Comment (azkyakhan) [inline src/components/Menu.vue:14]: Improve indentation

File: changelogs/unreleased/-1vju5j1.yml (hunk 1 above)
Comment (meet-aniket) [reply changelogs/unreleased/-1vju5j1.yml]: Removed sir

File: src/App.vue (hunk 2 above)
Comment (meet-aniket) [reply src/App.vue]: Removed sir

File: src/components/Menu.vue (hunk 3 above)
Comment (meet-aniket) [reply src/components/Menu.vue:27]: Used sir

File: src/components/Menu.vue (hunk 4 above)
Comment (meet-aniket) [reply src/components/Menu.vue:47]: Arranged sir

File: src/views/Settings.vue (hunk 6 above)
Comment (meet-aniket) [reply src/views/Settings.vue]: Used sir

File: changelogs/unreleased/-1vju5j1.yml (hunk 7 above)
Comment (meet-aniket) [reply changelogs/unreleased/-1vju5j1.yml:5]: Updated sir

File: changelogs/unreleased/-1vju5j1.yml [hunk 14]
```diff
@@ -0,0 +1,6 @@
+---
+title: Removed some bugs from application
```
[the remark this answers is not in the document]
Comment (meet-aniket) [reply changelogs/unreleased/-1vju5j1.yml]: now reverted

File: src/router/index.ts (hunk 8 above)
Comment (meet-aniket) [reply src/router/index.ts]: all are in single quote now

File: src/views/Completed.vue (hunk 10 above)
Comment (meet-aniket) [reply src/views/Completed.vue]: now arranged

File: changelogs/unreleased/-1vju5j1.yml (hunk 11 above)
Comment (meet-aniket) [reply changelogs/unreleased/-1vju5j1.yml]: now added ma'am

File: src/components/Menu.vue (hunk 12 above)
Comment (meet-aniket) [reply src/components/Menu.vue]: removed

File: src/components/Menu.vue (hunk 13 above)
Comment (meet-aniket) [reply src/components/Menu.vue:10]: improved

File: src/App.vue [hunk 15]
```diff
@@ -1,24 +1,29 @@
 <template>
   <ion-app>
-    <ion-router-outlet />
+    <ion-split-pane content-id="main-content" when="lg">
+      <Menu />
+      <ion-router-outlet id="main-content"></ion-router-outlet>
```
Comment (ymaheshwari1) [inline src/App.vue:5]: Use self closing tag

Comment (meet-aniket) [reply src/App.vue]: used sir

File: src/router/index.ts [hunk 16]
```diff
@@ -21,34 +23,47 @@ const loginGuard = (to: any, from: any, next: any) => {
   }
 };
 
+
```
Comment (ymaheshwari1) [inline src/router/index.ts:26]: Remove unwanted lines.

**Comment by adityasharma7** [comment]
Closing as lots of unwanted changes and blank commit messages

**Pull request closed without merging**